extension.ts

```diff
@@ -360,6 +360,8 @@ export async function activate(context: vscode.ExtensionContext) {
   }
 
   await Promise.all([
+    // This is in order to disable automatic keyboard navigation in lists
+    vscode.commands.executeCommand('setContext', 'listAutomaticKeyboardNavigation', false),
```

Comment:
We ended up wrapping this as we've found that `setContext` sometimes takes awhile to complete. It doesn't make much difference in this context, but to follow with the pattern, can you call:

```
  await VsCodeContext.Set('listAutomaticKeyboardNavigation', false);
```